.requirements

```diff
@@ -17,3 +17,5 @@ NGX_WASM_MODULE=a7087a37f0d423707366a694630f1e09f4c21728
 WASMER=3.1.1
 WASMTIME=14.0.3
 V8=10.5.18
+
+NGX_BROTLI=25f86f0bac1101b6512135eac5f93c49c63609e3 # v1.0.0rc
```

Comment:
This release version is really old, I wonder if we should pin to the `master` commit `a71f9312c2deb28875acc7bacfdd5695a111aa53` instead as it seems to contain a couple of fixes along the way.

Reply:
done!